src/locales/lang/en.ts

```diff
@@ -197,7 +199,7 @@ const msg = {
   dayCutTip: "Last 1 day",
   weekCutTip: "Last 1 week",
   monthCutTip: "Last 1 month",
-  serverZone: "Server Zone",
+  serverZone: "Time Zone",
```

Comment:
```suggestion
  serverZone: "OAP Server Time Zone",
```
Is this suitable?

Reply:
Yes. It's fine.

Reply:
Could you take a review locally to see the layout? Whether is it too long?

Reply:
> Could you take a review locally to see the layout? Whether is it too long?

I have updated the style.

<img width="651" alt="3" src="https://user-images.githubusercontent.com/20871783/161893204-2864ba87-6711-48f0-b5b1-92939eaee126.png">